.github/workflows/release-on-tag.yml

```diff
@@ -0,0 +1,25 @@
+name: Create new empty release
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  build:
+    name: Create Release
+    runs-on: ubuntu-latest
```

Comment:
nit: Can we change this to use Debian instead of Ubuntu?

Reply:
sadly no, only Ubuntu is supported right now, see: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idruns-on